cwms-data-api/src/main/java/cwms/cda/api/MeasurementController.java

```diff
@@ -0,0 +1,247 @@
+            } else {
+                dao.storeMeasurements(measurements, failIfExists);
+                ctx.status(HttpServletResponse.SC_CREATED).json("Created Measurements");
+            }
+        }
+    }
+
+    @OpenApi(ignore = true)
+    @Override
+    public void update(@NotNull Context ctx, @NotNull String locationId) {
+        try (final Timer.Context ignored = markAndTime(GET_ONE)) {
+            throw new UnsupportedOperationException(NOT_SUPPORTED_YET);
+        }
+    }
+
+    @OpenApi(
+            queryParams = {
+                    @OpenApiParam(name = OFFICE, required = true, description = "Specifies the office of the measurements to delete"),
+                    @OpenApiParam(name = MIN_NUMBER, description = "Specifies the min number-id of the measurement to delete."),
+                    @OpenApiParam(name = MAX_NUMBER, description = "Specifies the max number-id of the measurement to delete."),
+                    @OpenApiParam(name = MIN_DATE, description = "Specifies the minimum date (in ISO-8601 format) of the measurement to delete."),
+                    @OpenApiParam(name = MAX_DATE, description = "Specifies the maximum date (in ISO-8601 format) of the measurement to delete."),
+            },
+            description = "Delete an existing measurement.",
+            method = HttpMethod.DELETE,
+            tags = {TAG},
+            responses = {
+                    @OpenApiResponse(status = "204", description = "Measurement successfully deleted."),
+                    @OpenApiResponse(status = "404", description = "Measurement not found.")
+            }
+    )
+    @Override
+    public void delete(@NotNull Context ctx, @NotNull String locationId) {
+        String officeId = requiredParam(ctx, OFFICE);
+        String minNum = ctx.queryParam(MIN_NUMBER);
+        String maxNum = ctx.queryParam(MAX_NUMBER);
+        Instant minDate = parseInstant(ctx.queryParam(MIN_DATE));
+        Instant maxDate = parseInstant(ctx.queryParam(MAX_DATE));
+        try (Timer.Context ignored = markAndTime(DELETE)) {
+            DSLContext dsl = getDslContext(ctx);
+            MeasurementDao dao = new MeasurementDao(dsl);
+            dao.deleteMeasurements(officeId, locationId, minDate, maxDate, UnitSystem.EN.getValue(), null,
+                    null, null, null, minNum, maxNum, null, null);
+            ctx.status(HttpServletResponse.SC_NO_CONTENT).json( "Measurements for " + locationId + " Deleted");
+        }
+    }
+
+    private Instant parseInstant(String date) {
+        Instant retVal = null;
+        if(date != null && !date.isEmpty()) {
+            retVal = Instant.parse(date);
+        }
+        return retVal;
+    }
+
+    static List<Measurement> parseMeasurements(@NotNull Context ctx, ContentType contentType) {
+        List<Measurement> measurements;
+        try {
+            measurements = Formats.parseContentList(contentType, ctx.body(), Measurement.class);
+        } catch (FormattingException e) {
```

Comment:
This should throw an error to the client.

Reply:
Updated to throw error to client now that only lists are supported